factcast-schema-registry-cli/src/main/resources/site/hugo-theme-docdock-master/exampleSite/content/shortcode/excerpt-include.md

```diff
@@ -0,0 +1,27 @@
++++
+draft = false
+title = "excerpt-include"
+description = ""
+
+[menu.main]
+parent = "excerpt"
+identifier = "excerpt-include"
++++
+
+The Excerpt Include shortcode is used to display 'excerpted' (that is, a segment of) content from one page in another.
+Before you can use this shortcode, the excerpt must have been defined using the Excerpt shortcode. {{%alert info%}}Note that you can have more than one Excerpt Include shortcode on a page (although you can have only one Excerpt shortcode on a page).{{%/alert%}}
+
+
+## Usage
+
+| Parameter | Default | Description |
+|:--|:--|:--|
+| filename | **required** | Type the filename of the page that contains the excerpt to be displayed.<br/>Path is relative to the content folder|
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[table-cell-padding] Cell should be padded](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4955738)